examples/bare_sphere/run_sphere.py

```diff
@@ -0,0 +1,290 @@
+# Optimization paramenters: user input required
+import re
+import sys
+import math
+import pandas as pd
+import openmc
+import pdb
+import matplotlib.pyplot as plt
+from togamgxs.main import TOGA
+from togamgxs.groups import Groups
+from collections import OrderedDict
+import os
+
+sphere_size = sys.argv[1]
```

Comment:
I tried running run_sphere.py but I might be going about it wrong. Do I just run run_sphere.py directly? I have an empty sys.argv, so it makes me think I need to pass some arguments somehow. Let me know how to run this example.